pkgs/by-name/tf/tfctl/package.nix

```diff
@@ -0,0 +1,71 @@
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  fetchFromGitHub,
+  installShellFiles,
+  kubectl,
+  writeScript,
+}:
+let
+  commitHash = "a53488094851773978301511922c2ced1397cdf9"; # matches tag release
+  shortCommitHash = builtins.substring 0 8 commitHash;
+in
```

Comment:
I would not recommend this and rather just replace the buildsha with the tag ref or some dummy value to make auto updates easier.